sdk/identity/Azure.Identity/src/UnsafeTokenCacheOptions.cs

```diff
@@ -14,24 +15,22 @@ public abstract class UnsafeTokenCacheOptions : TokenCachePersistenceOptions
         /// <summary>
         /// The delegate to be called when the Updated event fires.
         /// </summary>
-        /// <value></value>
         protected internal abstract Task TokenCacheUpdatedAsync(TokenCacheUpdatedArgs tokenCacheUpdatedArgs);
 
         /// <summary>
         /// Returns the bytes used to initialize the token cache. This would most likely have come from the <see cref="TokenCacheUpdatedArgs"/>.
-        /// This implementation will get called by the default implementation of <see cref="RefreshCacheAsync(TokenCacheNotificationDetails)"/>.
-        /// It is recommended to provide an implementation for <see cref="RefreshCacheAsync(TokenCacheNotificationDetails)"/> rather than this method.
+        /// This implementation will get called by the default implementation of <see cref="RefreshCacheAsync(TokenCacheRefreshArgs, CancellationToken)"/>.
+        /// It is recommended to provide an implementation for <see cref="RefreshCacheAsync(TokenCacheRefreshArgs, CancellationToken)"/> rather than this method.
         /// </summary>
-        /// <value></value>
         protected internal abstract Task<ReadOnlyMemory<byte>> RefreshCacheAsync();
 
         /// <summary>
         /// Returns the bytes used to initialize the token cache. This would most likely have come from the <see cref="TokenCacheUpdatedArgs"/>.
         /// It is recommended that if this method is overriden, there is no need to provide a duplicate implementation for the parameterless <see cref="RefreshCacheAsync()"/>.
         /// </summary>
-        /// <param name="details"></param>
-        /// <returns></returns>
-        protected internal virtual async Task<TokenCacheDetails> RefreshCacheAsync(TokenCacheNotificationDetails details) =>
+        /// <param name="details">The <see cref="TokenCacheRefreshArgs"/> containing information about the current state of the cache.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> controlling the lifetime of this operation.</param>
+        protected internal virtual async Task<TokenCacheData> RefreshCacheAsync(TokenCacheRefreshArgs details, CancellationToken cancellationToken = default) =>
```

Comment:
nit: should the parameter name be called `args` or `refreshArgs` now that we changed the type name?